### ISSUE TITLE
help on error : cannot import name 'LLaMAConfig' from 'transformers'

### ISSUE THREAD

**Issue by Paolo-Oliverio** [underlines/awesome-marketing-datascience#2]
I followed the guide for native windows with no errors but at start I get this error:

```
(textgen) C:\Users\paolo\ai\text-generation-webui>python server.py --model llama-13b-hf --load-in-4bit --no-stream
Warning: --load-in-4bit is deprecated and will be removed. Use --gptq-bits 4 instead.

Loading llama-13b-hf...
CUDA extension not installed.
Traceback (most recent call last):
  File "C:\Users\paolo\ai\text-generation-webui\server.py", line 197, in <module>
    shared.model, shared.tokenizer = load_model(shared.model_name)
  File "C:\Users\paolo\ai\text-generation-webui\modules\models.py", line 94, in load_model
    model = load_quantized(model_name)
  File "C:\Users\paolo\ai\text-generation-webui\modules\GPTQ_loader.py", line 55, in load_quantized
    model = load_quant(str(path_to_model), str(pt_path), shared.args.gptq_bits)
  File "C:\Users\paolo\ai\text-generation-webui\repositories\GPTQ-for-LLaMa\llama.py", line 220, in load_quant
    from transformers import LLaMAConfig, LLaMAForCausalLM
ImportError: cannot import name 'LLaMAConfig' from 'transformers' (C:\Users\paolo\miniconda3\envs\textgen\lib\site-packages\transformers\__init__.py)
```
Please help me sort this out.

**Comment by underlines** [comment]
it looks like you use the original transformer's implementation, instead of the fork that uses the @llama_push . can you check that? try `pip uninstall transformers` and then pip installing the one with `pip install git+https://github.com/zphang/transformers.git@llama_push`

**Comment by St33lMouse** [comment]
It's been a few days and I've seen many errors since then, but I think I solved this one because the capitalization of LLaMA was different from one of the things referring to it, where was Just LlamaConfig.  I'm sorry I don't remember more. But I'm pretty sure this one was a shift in capitalization somewhere.

**Comment by ysyecust** [comment]
When it will merge into master branch?

**Comment by ysyecust** [comment]
When it will be merged into master branch?

**Comment by ashkan-leo** [comment]
Getting the same error. I installed rev = "68d640f7c368bcaaaecfc678f11908ebbd3d6176" which is supposed to include all the llama specific stuff... wondering how I can unblock

**Comment by qianshengzj** [comment]
> it looks like you use the original transformer's implementation, instead of the fork that uses the @llama_push . can you check that? try `pip uninstall transformers` and then pip installing the one with `pip install git+https://github.com/zphang/transformers.git@llama_push`


mark, I solved this problem

**Comment by bobqianic** [comment]
Just use ```pip install https://github.com/huggingface/transformers/archive/refs/heads/main.zip```

**Comment by OrtalAshkenaziGong** [comment]
I use transformers version 4.29.0.dev0
and I get the same error 

from transformers import AutoTokenizer, LLaMAForCausalLM
model = LLaMAForCausalLM.from_pretrained('decapoda-research/llama-7b-hf')
ImportError: cannot import name 'LLaMAForCausalLM' from 'transformers'

**Comment by underlines** [comment]
rewrote the whole guide. issue should be solved in the meantime

**Comment by AsteriaCao** [comment]
You can import LLaMAConfig like this: from transformers.models.llama.configuration_llama import LlamaConfig. 
BTW, my transformers version is 4.45.0.dev0.
